drivers/hdlc_rcp_if/Kconfig.nxp

```diff
@@ -0,0 +1,9 @@
+# Openthead HDLC RCP communication Interface configuration options
+# Copyright 2024 NXP
+# All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
BSD license should be changed to `Apache-2.0`

Reply:
I will change the licences regarding all your comments.

Reply:
Note that you can only change the license if you are the author. If you took the code from somewhere else, then you need to use the original license.

Reply:
resolved